calico-cloud/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-description: Learn about policies, tiers, and policy evaluation.
+description: Understand how tiered policy works and supports microsegmentation.
 ---
 
-# Policy tiers tutorial
+# Get started with policy tiers
```

Comment:
The above changes happened by syncing enterprise version with cloud version. It make sense to sync these, unless there is reason behind the diff.